devops/Makefile

```diff
@@ -181,4 +306,107 @@ deploy-web:
+	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMIO_BPM_NAME) --wait
+
+build-forms-flow-api:
+	test -n "$(BUILD_NAMESPACE)"
+	test -n "$(FORMIO_API_NAME)"
+	test -n "$(SOURCE_REPOSITORY_REF)"
+	test -n "$(SOURCE_CONTEXT_DIR)"
+	test -n "$(FORM_AI_BUILD_TEMPLATE_PATH)"
+	@echo "+\n++ BUILDING $(FORMIO_API_NAME)"
+	@echo "+\n++ BUILDING $(FORMIO_API_NAME) DIR: $(SOURCE_CONTEXT_DIR)$(FORMIO_API_NAME) BuildFile:$(FORM_AI_BUILD_TEMPLATE_PATH)\n+"
+	@oc -n $(BUILD_NAMESPACE) process -f $(FORM_AI_BUILD_TEMPLATE_PATH) -p NAME=$(FORMIO_API_NAME) -p TAG=$(BUILD_TAG) -p FORMIO_SOURCE_REPO_URL=$(FORMIO_SOURCE_REPO_URL) -p FORMIO_SOURCE_REPO_BRANCH=$(FORMIO_SOURCE_REPO_BRANCH) -p SOURCE_REPOSITORY_REF=$(SOURCE_REPOSITORY_REF) -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)$(FORMIO_API_NAME)| oc -n $(BUILD_NAMESPACE) apply -f -
+	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMIO_API_NAME) --wait
+
+build-forms-flow-web:
+	test -n "$(BUILD_NAMESPACE)"
+	test -n "$(FORMIO_WEB_NAME)"
+	test -n "$(SOURCE_REPOSITORY_REF)"
+	test -n "$(SOURCE_CONTEXT_DIR)"
+	test -n "$(FORM_AI_BUILD_TEMPLATE_PATH)"
+	@echo "+\n++ BUILDING $(FORMIO_WEB_NAME)"
+	@echo "+\n++ BUILDING $(FORMIO_WEB_NAME) DIR: $(SOURCE_CONTEXT_DIR)$(FORMIO_WEB_NAME) BuildFile:$(FORM_AI_BUILD_TEMPLATE_PATH)\n+"
+	@oc -n $(BUILD_NAMESPACE) process -f $(FORM_AI_BUILD_TEMPLATE_PATH) -p NAME=$(FORMIO_WEB_NAME) -p TAG=$(BUILD_TAG) -p FORMIO_SOURCE_REPO_URL=$(FORMIO_SOURCE_REPO_URL) -p FORMIO_SOURCE_REPO_BRANCH=$(FORMIO_SOURCE_REPO_BRANCH) -p SOURCE_REPOSITORY_REF=$(SOURCE_REPOSITORY_REF) -p SOURCE_CONTEXT_DIR=$(SOURCE_CONTEXT_DIR)$(FORMIO_WEB_NAME)| oc -n $(BUILD_NAMESPACE) apply -f -
+	@oc -n $(BUILD_NAMESPACE) start-build bc/$(FORMIO_WEB_NAME) --wait
+
+deploy-ha-mongo:
+	test -n "$(NAMESPACE)"
+	@oc -n $(NAMESPACE) process -f $(HAMONGO_DB_TEMPLATE_PATH) --param-file=$(HAMONGO_DB_PARAM_PATH) | oc -n $(NAMESPACE) apply -f -
+
+deploy-forms-flow-db-secrets:
+	test -n "$(NAMESPACE)"
+	@oc -n $(NAMESPACE) process -f $(SECRETS_PATRONI_DB_TEMPLATE_PATH) | oc -n $(NAMESPACE) apply -f -
+
+create-ffa-bpm-db:
+	test -n $(NAMESPACE)
+	@echo "+\n++ Creating DB (FFA_BPM_DB) in $(NAMESPACE) patroni db cluster"
+	@oc -n $(NAMESPACE) process -f $(CREATE_FFA_DB_TEMPLATE_PATH) -p DB_NAME=$(FFA_BPM_DB_NAME) -p BUILD_NAMESPACE=$(BUILD_NAMESPACE) -p JOB_NAME=bpm | oc -n $(NAMESPACE) apply -f -
+
+create-ffa-api-db:
+	test -n $(NAMESPACE)
+	@echo "+\n++ Creating DB (FFA_API_DB) in $(NAMESPACE) patroni db cluster"
+	@oc -n $(NAMESPACE) process -f $(CREATE_FFA_DB_TEMPLATE_PATH) -p DB_NAME=$(FFA_API_DB_NAME) -p BUILD_NAMESPACE=$(BUILD_NAMESPACE) -p JOB_NAME=api | oc -n $(NAMESPACE) apply -f -
+
+deploy-forms-flow-secrets:
+	test -n "$(NAMESPACE)"
+	@oc -n $(NAMESPACE) process -f $(SECRETS_TEMPLATE_PATH) --param-file=$(SECRETS_PARAM_PATH) | oc -n $(NAMESPACE) apply -f -
+
+deploy-forms-flow-forms:
+	test -n "$(NAMESPACE)"
+	test -n "$(BUILD_NAMESPACE)"
+	test -n "$(BUILD_REF)"
```

Comment:
It seems that BUILD_REF and HOST_PREFIX are not being used. Can you please check it and remove them if possible?